apps/projection/src/app/component/city-card/city-card.component.ts

```diff
@@ -1,13 +1,47 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit, ViewEncapsulation, inject } from '@angular/core';
+import { CardComponent } from '../../ui/card/card.component';
+import { City } from '../../model/city.model';
+import { CityStore } from '../../data-access/city.store';
+import {
+  FakeHttpService,
+  randomCity,
+} from '../../data-access/fake-http.service';
 
 @Component({
   selector: 'app-city-card',
-  template: 'TODO City',
+  template: `<app-card
+    [list]="cities"
+    [addOneStore]="store.addOne.bind(store)"
+    [removeStore]="store.deleteOne.bind(store)"
+    [addRandom]="addRandom"
+    customClass="bg-light-orange"
+    label="name"></app-card>`,
+  styles: [
+    `
+      .bg-light-orange {
+        background-color: rgba(250, 100, 0, 0.1);
+      }
+      .bg-light-orange > .imgHolder {
+        content: url('./../../../assets/img/city.jpg');
+      }
+    `,
+  ],
+  encapsulation: ViewEncapsulation.None,
   standalone: true,
-  imports: [],
+  imports: [CardComponent],
 })
 export class CityCardComponent implements OnInit {
-  constructor() {}
+  cities: City[] = [];
 
-  ngOnInit(): void {}
+  http: FakeHttpService = inject(FakeHttpService);
+  store: CityStore = inject(CityStore);
+  addRandom: () => City = randomCity;
+
+  ngOnInit(): void {
+    this.store.deleteOne;
+    this.http.fetchCities$.subscribe((c) => {
+      this.store.addAll(c);
+      this.store.citys$.subscribe((c) => (this.cities = c));
+    });
```

Comment:
nested subscribe should be avoided at all cost, 

prefer a declarative approach.